lectures/introduction_dynamics/linear_models.md

```diff
@@ -1235,18 +1235,18 @@ using LaTeXStrings, QuantEcon, Plots
 ```{code-cell} julia
 ϕ0, ϕ1, ϕ2 = 1.1, 0.8, -0.8
 
-A = [1.0   0.0   0
-        ϕ0    ϕ1    ϕ2
-        0.0   1.0   0.0]
+A = [1.0 0.0 0
+     ϕ0 ϕ1 ϕ2
+     0.0 1.0 0.0]
 C = zeros(3, 1)
 G = [0.0 1.0 0.0]
 μ_0 = ones(3)
 
-lss = LSS(A, C, G; mu_0=μ_0)
+lss = LSS(A, C, G; mu_0 = μ_0)
```

Comment:
After unicode conversion this becomes `lss = LSS(A, C, G; mu_0)`   Note that we don't need to repeat.